packages/ui/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,104 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/outline';
+import type { Culture, Newsroom, NewsroomCompanyInformation } from '@prezly/sdk';
+import { hasAnySocialMedia } from '@prezly/theme-kit-core';
+
+import { Link } from '../Link';
+import { SocialMedia } from '../StoryShareLinks';
+
+import { SubFooter } from './SubFooter';
+
+export interface Props {
+    className?: string;
+    companyInformation: NewsroomCompanyInformation;
+    externalSiteLink?: string;
+    hasStandaloneAboutPage?: boolean;
+    hasStandaloneContactsPage?: boolean;
+    publicGalleriesCount: number;
+    locale: Culture['code'];
+    newsroom: Pick<
+        Newsroom,
+        'uuid' | 'custom_data_request_link' | 'display_name' | 'is_white_labeled'
+    >;
+}
+
+export function Footer({
+    className,
+    companyInformation,
+    externalSiteLink,
+    hasStandaloneAboutPage,
+    hasStandaloneContactsPage,
+    locale,
+    newsroom,
+    publicGalleriesCount,
+}: Props) {
+    const hasSocialMedia = hasAnySocialMedia(companyInformation);
+
+    return (
+        <footer className={className}>
+            <div className="p-12 flex flex-col md:flex-row bg-gray-900 justify-between">
```

Comment:
This is partly a design question, but shouldn't the header and footer contents have a max-width? (i.e. wrapped in a container)

Reply:
@iamhugofelix can you advice here